pinc/CharSuites.inc

```diff
@@ -179,44 +179,55 @@ foreach($charsuite_files as $charsuite_file)
     include_once($charsuite_file);
 }
 
-function get_pickersets($charsuites)
+trait CharSuiteSet
 {
-    $pickersets = [];
-    foreach($charsuites as $charsuite)
+    public function get_pickersets()
     {
-        $pickersets[] = $charsuite->pickerset;
+        $pickersets = [];
+        foreach($this->get_charsuites() as $charsuite)
+        {
+            $pickersets[] = $charsuite->pickerset;
+        }
+        return $pickersets;
+    }
+
+    public function get_valid_codepoints()
+    {
+        # Codepoints applicable to all projects
+        $global_codepoints = [
+            'U+000A', # new lines
+            'U+000D', # carriage returns
+        ];
+
+        $codepoints = $global_codepoints;
+        foreach($this->get_charsuites() as $charsuite)
+        {
+            $codepoints = array_merge($codepoints, $charsuite->codepoints);
+        }
+        return $codepoints;
     }
-    return $pickersets;
 }
 
-function get_valid_codepoints($charsuites)
+class QuizProject
```

Comment:
This should be in its own `pinc/QuizProject.inc` file. And QuizProject doesn't make much sense -- why not just call it `Quiz` and, in the future, we can move other quiz-related stuff in here.

Reply:
There is already a Quiz class and Quiz.inc. I moved it to Quiz.inc and called it QuizCharSuites.